xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1221,30 +1310,43 @@ struct AppEnvironment {
             gatewayURL: defaultGateway,
             sphereIdentity: defaultSphereIdentity
         )
+        self.noosphere = noosphere
```

Comment:
Keep Noosphere around. Now that services are "flattened" we need to make sure we have one property for each service.